### PR TITLE
[ADD] new module crm_claim_product_supplier to filled supplier and in…

### PR DESCRIPTION
**Pull request by yaniaular**
…voice supplier in claim line. new module crm_claim_product_supplier to filled supplier and invoice supplier in claim line

### CODE REVIEW

File: crm_claim_product_supplier/model/crm_claim_rma.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+
+from openerp import models, fields, api
+
+
+class claim_line(models.Model):
```
Comment (pedrobaeza) [inline crm_claim_product_supplier/model/crm_claim_rma.py]: CamelCase

File: crm_claim_product_supplier/model/crm_claim_rma.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+            self.supplier_id = self.prodlot_id.supplier_id.id
+
+            # Take all stock moves with incoming type of operation
+            sm_receipts = claim_obj._get_stock_moves_with_code()
+
+            # Get traceability of serial/lot number
+            quant_obj = self.env['stock.quant']
+            quants = quant_obj.search([('lot_id', '=', self.prodlot_id.id)])
+            moves = set()
```
Comment (pedrobaeza) [inline crm_claim_product_supplier/model/crm_claim_rma.py]: Do `moves = quant.mapped('history_ids')` and use recordsets

Comment (nhomar) [inline crm_claim_product_supplier/model/crm_claim_rma.py]: 2015-08-05 15:23 GMT-05:00 Pedro M. Baeza notifications@github.com:

> Do moves = quant.mapped('history_ids') and use recordsets

+1

## 

---

Saludos Cordiales

CEO at Vauxoo https://www.vauxoo.com Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
http://about.me/nhomar?promo=email_sig

File: crm_claim_product_supplier/model/crm_claim_rma.py (hunk 1 above)
Comment (nhomar) [inline crm_claim_product_supplier/model/crm_claim_rma.py]: 2015-08-05 15:03 GMT-05:00 Pedro M. Baeza notifications@github.com:

> CamelCase

+1

## 

---

Saludos Cordiales

CEO at Vauxoo https://www.vauxoo.com Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
http://about.me/nhomar?promo=email_sig

**Comment by gurneyalex** [comment]
needs rebase

**Comment by osvalr** [comment]
@KarenKawaii can you help me with this travis configuration? it seems old dated

**Comment by yaniaular** [comment]
LGTM :+1:

**Comment by osvalr** [comment]
Hi @pedrobaeza  @gurneyalex rebase is done, can you review this please?

File: crm_claim_product_supplier/README.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+CRM Claim Product Supplier
```
Comment (pedrobaeza) [inline crm_claim_product_supplier/README.rst:6]: Put upper line

File: crm_claim_product_supplier/models/claim_line.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+from openerp import api, fields, models
+
+
+class ClaimLine(models.Model):
+
+    _inherit = 'claim.line'
+
+    supplier_id = \
+        fields.Many2one('res.partner', string='Supplier',
+                        compute='_compute_supplier_and_supplier_invoice',
```
Comment (pedrobaeza) [inline crm_claim_product_supplier/models/claim_line.py]: Why not a simply related field?:

```
fields.Many2one(comodel_name='res.partner', related='prodlot_id.supplier_id')
```

**Comment by osvalr** [comment]
@nhomar review this ASAP

**Comment by nhomar** [comment]
:+1:

**Pull request merged**